Makefile

```diff
@@ -1,27 +1,37 @@
-AUTOTAGS := $(shell ./btrfs_tag.sh) $(shell ./btrfs_installed_tag.sh) $(shell ./libdm_tag.sh) $(shell ./ostree_tag.sh) $(shell ./selinux_tag.sh)
-TAGS ?= seccomp
+SELINUXTAG := $(shell ./selinux_tag.sh)
+STORAGETAGS := $(shell ./btrfs_tag.sh) $(shell ./btrfs_installed_tag.sh) $(shell ./libdm_tag.sh) $(shell ./ostree_tag.sh)
+SECURITYTAGS ?= seccomp $(SELINUXTAG)
```

Comment:
I am afraid that removing `TAGS` could break some builds in the wild that use `TAGS`. To remain backwards compat, we could add:
```
TAGS ?= $(SECURITYTAGS) $(STORAGETAGS)
[...]
BUILDFLAGS := -tags "$(TAGS)"
```

Would this work or am I missing something?

Reply:
Added.